program-runtime/src/invoke_context.rs

```diff
@@ -280,10 +280,6 @@ impl<'a> InvokeContext for ThisInvokeContext<'a> {
             }
             self.current_compute_budget = compute_budget;
 
-            if !self.feature_set.is_active(&tx_wide_compute_cap::id()) {
-                self.compute_meter = ComputeMeter::new_ref(self.current_compute_budget.max_units);
```

Comment:
This resets the compute meter for each instruction, without this we would be tracking the meter tx-wide.  I think in order to track tx wide but cap per instruction we would need to track it independently

Reply:
Looks like compute units are accumulated to `timings` per instruction (https://github.com/solana-labs/solana/blob/master/runtime/src/message_processor.rs#L121-L125). Though this reset causes `post_remaining_units` to be resolved into `max_units` again, maybe to get `post_remaining_units` differently?